### PR TITLE
SiteToCloud VPN_User_Story

### PR DESCRIPTION
**Pull request by DanXu-ChinaTelecom**
#### What type of PR is this?

Add one of the following kinds:

* documentation

#### What this PR does / why we need it:

* user story to explain commerical needs

### CODE REVIEW

**Comment by ShutingQing** [comment]
Dan, @DanXu-ChinaTelecom Dan, let's create each Issue, and linked to each PR.

File: documentation/API_documentation/S2CVPN_User_Story.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# SiteToCloud(S2C)  VPN API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an application developer or industry customer, I want to request (using my application server/backend service)cloud leased network connection service between siteA/cloudA and  siteB/cloudB from a Communication Service Provider (CSP), so that I can ensure better and more reliable quality for our end users at any required time with proper cost. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User<br> **Actors:** Application service providers,  industry customer,  application developers.<br> **Scope:** Service Booking(check network availability level, resource availability and corresponding fees )   \- Create/Remove/Modify/Query  service instance (S2C VPN session resource) |
```
Comment (ShutingQing) [inline documentation/API_documentation/S2CVPN_User_Story.md]: For customer here, should we specify more detialed?

Comment (ShutingQing) [inline documentation/API_documentation/S2CVPN_User_Story.md]: For Scope here, do you think it's better to claify in usage of Site to Cloud VPN Service?

File: documentation/API_documentation/S2CVPN_User_Story.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.</li><li>The Customer:Administrator has onboarded the Customer:User to the platform.</li><li>The Customer either owns the device subscriptions or has agreed to the terms and conditions of the CSP for managing consent of subscription owners.</li><li>The means to get the access token are known to the Customer:User to ensure secure access of the API.|
```
Comment (ShutingQing) [inline documentation/API_documentation/S2CVPN_User_Story.md:7]: I'll suggest that we'd better have a flow diagraph to better elaborate the relationship of the roles here.

**Review by ShutingQing — COMMENTED** [review]
Detailed comments are marked in "File Changed". 

For Pre-condition Roles here. I feel like it's kind of a bit complex. I'll suggest maybe later we can have a diagrph to better elaborate the relationship and the work flow.

File: documentation/API_documentation/S2CVPN_User_Story.md (hunk 1 above)
Comment (DanXu-ChinaTelecom) [reply documentation/API_documentation/S2CVPN_User_Story.md]: Yes，agree

**Review by ShutingQing — APPROVED** [review]

**Pull request merged**